ocaml/parsing/parser.mly

```diff
@@ -4263,6 +4341,45 @@ strict_function_or_labeled_tuple_type:
    | { Mode.empty }
    | mode_expr_legacy_nonempty {$1}
 ;
+
+/* New mode annotation, followed by AT or ATAT and thus enjoys its own namespace */
```

Comment:
```suggestion
/* New mode annotation, introduced by AT or ATAT */
```

Reply:
I don't think future developers will wonder that. The fact that people can write mode aliases should make that clear.